ingest/scripts/heuristic_group_segments.py

```diff
@@ -211,6 +211,11 @@ def main(
                 if segment in group
             ]
         )
+        if not joint_key:
```

Comment:
Let's fix this at the root, filter out non-matching much earlier

Reply:
True this could be filtered out in `parse_nextclade_sort_output.py` - I will try to do that there